extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -261,8 +249,10 @@ public function setMemcache( $memcKey, $data, $loggingParams ) {
 	 * @return string
 	 */
 	public function getMemcKey( $params ) {
-		$geo = empty( $params['geo'] ) ? 'ALL' : $params['geo'];
-		$memcKey = wfMemcKey( 'monetization_module', $params['cache'], $geo, $params['max'] );
+		$geo = empty( $params['geo'] ) ? self::COUNTRY_CODE_ALL : $params['geo'];
+		$fromSearch = empty( $params['from_search'] ) ? 'direct' : 'search';
+		$adEngine = empty( $params['ad_engine'] ) ? 'mon' : 'ad';
+		$memcKey = wfMemcKey( 'monetization_module', $params['cache'], $geo, $params['max'], $fromSearch, $adEngine );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$memcKey". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)